recycling-server/db/migrate/20170302234459_create_categories.rb

```diff
@@ -4,6 +4,12 @@ def change
       t.string :name, null: false
       t.text :description
       t.string :image_link
+      t.string :recycle_class # Five digit string, "abcde"
```

Comment:
Do we really want to use letters to represent categories? This seems arbitrary and error-prone. I would suggest having another table for the recycle class so that it's more obvious. Or probably directly use the full name instead of single-character representation.